### DISCUSSION TITLE
Open target file in Markor after share into

### DISCUSSION THREAD

**Discussion by GlassGruber** [gsantner/markor#2150] [Q&A] [answered]
I've recently updated Markor from a quite older version (2.8 I think) and now when I share into a file, the target file is not opened right after anymore. 
Is there some settings to turn this behavior on back again?
Thank you!

**Comment by harshad1** [comment, marked as answer]
I believe I fixed the bug causing this change recently. It should be fixed in the next release.

**Reply by GlassGruber** [reply]
Oh nice, I thought it was "a feature, not a bug"™.  🙃 

I think it could actually be a desirable ui option in the share into view, like a checkbox for "open target file after share" or similar. Maybe with a setting somewhere to make it default checked/unchecked.

**Reply by harshad1** [reply]
We try pretty hard to minimise settings bloat. Files open _fast_. So they opening it automatically is minimally problematic.
